src/main/java/com/obj/nc/config/NcAppConfigProperties.java

```diff
@@ -47,4 +47,16 @@ public class NcAppConfigProperties {
     @Value("${nc.app.url.context-path:/notiflow}")
     private String contextPath;
 
+    private Web web = new Web();
```

Comment:
tuto strukturu som nepochopil..

Zda sa ze chces mat akysi default, resp setting,.. preco to ale je zabalene v triede Web? preco to neni simple properties ako ine settingy

Reply:
mozu byt aj simple, ale ked sa properties takto zabalia do podtried, a pribudli by tam dalsie tak potom pri definovani nejakej metody co ich potrebuje jej mozem ako argument poslat podtriedu namiesto triedy so vsetkymi properties alebo namiesto x parametrov

Reply:
ok, ale 1) nazov triedy Web je fakt divny,.. vobec mi neindikuje nic co by malo suvisiet s pagingom. Ono trieda Web mi neindikuje nic, lebo to je taky siroky pojem ze to moze byt hosiso.
2) ty vies spravit specialnu config triedu ktora ma tieto parametre a namapovat ich z prop filu..  napt nevedel by si proste mat config triedu

@Configuration
@ConfigurationProperties("nc.app.paging")
public class PagingProperties {
    
// aj pre tieto props vies setnut default
    int maxPageSize
    ...

    // standard getters and setters
}

Reply:
ok hodil som tie properties do samostatnej triedy